### PR TITLE
Fix/daef 166 strange artefacts ghost boxes caused by chromium issue

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR fixes strange artefacts/ghost boxes on loading screen.

### CODE REVIEW

File: app/components/loading/Loading.scss [hunk 1]
```diff
@@ -30,13 +35,13 @@
...
   .headline {
     overflow: visible;
     @include animated-ellipsis($width: 16px);
   }
 }
 
-.syncing {
-  color: #5e6066;
+.component, .headline, .logo {
+  -webkit-backface-visibility: hidden;
```
Comment (DominikGuzei) [inline app/components/loading/Loading.scss]: So i guess this is the fix?

**Comment by DominikGuzei** [comment]
@nikolaglumac how can i test this PR?

**Comment by nikolaglumac** [comment]
@DominikGuzei this issue happens only on certain GPUs / OS combinations. If you have never before seen this issue than there is no way you can test it :(
I know @darko-mijic reported he has seen this problem. I hope he will be able to test it...

**Comment by DominikGuzei** [comment]
@nikolaglumac ok yeah, i actually had this issue but it does not always happen … will try it on master to see if it's there so i can test if the fix works here 😞

**Comment by DominikGuzei** [comment]
Ok i cannot reproduce the error … so @darko-mijic should take a look if he can verify it

File: app/components/loading/Loading.scss (hunk 1 above)
Comment (nikolaglumac) [inline app/components/loading/Loading.scss]: Hopefully yes :)

**Comment by nikolaglumac** [comment]
🎉

**Comment by darko-mijic** [comment]
Before:
screen shot 2017-04-26 at 10 28 12

After:
screen shot 2017-04-26 at 10 28 06

**Pull request merged**